src/helpers/amazon-q/amazon-q-client.ts

```diff
@@ -49,17 +74,54 @@ export const callClient = async (
     conversationId: string;
     parentMessageId: string;
   }
-): Promise<ChatSyncCommandOutput> => {
+): Promise<ChatSyncCommandOutput | Error> => {
+  try {
+  message = truncateMessageIfNeeded(message);
```

Comment:
I think there's something wrong with indentation in try block here and in next method

Reply:
Ack. Fixed the indentation.